src/main/java/dev/galacticraft/mod/GalacticraftClient.java

```diff
@@ -24,12 +24,12 @@
 
 import dev.galacticraft.mod.block.GalacticraftBlock;
 import dev.galacticraft.mod.client.gui.screen.ingame.*;
-import dev.galacticraft.mod.client.model.MachineBakedModel;
```

Comment:
Can you remove all the unrelated changes in the file?

Reply:
one sec